dotnet/src/SemanticKernel/AI/OpenAI/Clients/OpenAIClientAbstract.cs

```diff
@@ -207,6 +207,18 @@ internal OpenAIClientAbstract(ILogger? log = null, IDelegatingHandlerFactory? ha
         }
     }
 
+    /// <summary>
+    /// Performs prompt normalization
+    /// </summary>
+    /// <param name="prompt">Prompt to normalize</param>
+    protected void NormalizePrompt(string prompt)
```

Comment:
method could be static, and even an extension method (i.e. `public static string NormalizeLineEndings(this string)`) in here:
https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/SemanticKernel/Text/StringExtensions.cs